package.json

```diff
@@ -95,12 +97,17 @@
     "test-electron": "browserify dist/test/electron.js | testron | faucet",
     "start": "budo example/main.js:example/bundle.js --live --open --port 3000 --css example/main.css --title 'deck.gl' --watch-glob '**/*.{html,css,js,glsl}' -- -t babelify -t envify"
   },
-  "standard": {
-    "ignore": [
-      "**/build/**",
-      "**/dist/**",
-      "**/examples/**",
-      "**/Documents/**"
-    ]
+  "eslintConfig": {
```

Comment:
Not strong reason, except that I like a clean root directory (without a lot of little files), and now the eslint config is so compact that it became reasonable to put it inside package.json. Easy to revert in case of any issues.